estate_account/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+{
+    'name': 'Real Estate Account',
+    'version': '0.0',
+    'category': 'Real Estate',
+    'summary': 'Manage your Real Estate accounts',
+    'license': 'LGPL-3',
+    'application': True,
+    'installable': True,
+    'depends': ['base', 'estate', 'account'],
```

Comment:
```suggestion
    'depends': ['estate', 'account'],
```
This is redundant as your `estate` module already relies on `base` module